### DISCUSSION TITLE
为什么我出来的效果非常糟糕呢？

### DISCUSSION THREAD

**Discussion by peterlau0626** [ymcui/Chinese-LLaMA-Alpaca#162] [Q&A]
![QQ截图20230415101417](https://user-images.githubusercontent.com/50452445/232178310-e7b18d7a-dee5-48ce-b223-1962c3257dc4.png)

我先把facebook原始模型转成hf格式，然后再使用merge_llama_with_chinese_lora_to_hf.py合并模型，两个模型都是13B的。最后使用transformers做推理。好像看起来非常糟糕。

**Comment by airaria** [comment]
貌似加载的是LLaMA模型？对话问答的话用Alpaca模型体验会更好。

**Comment by ninehills** [comment]
7B Alpaca 模型，效果也十分糟糕，基本处于不可用的情况。13B之后测试。

<img width="302" alt="image" src="https://user-images.githubusercontent.com/270298/233844755-38099f7f-6bd0-4af0-8237-ef11be00c75c.png">

**Reply by LiemLin** [reply]
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/16920347/d0c00a36-6cb9-423b-a463-c52a0cc12213)

效果还可以的，你是不是没用alpaca prompt模板

**Comment by ninehills** [comment]
<img width="855" alt="image" src="https://user-images.githubusercontent.com/270298/233849005-f9089c6a-ca36-47ad-94d9-497074475b38.png">

额，这个效果一言难尽。和会不会关系不大，感觉对中文的指令理解有点错乱。用的网友提供的merge后的13B模型。

**Reply by mshmoon** [reply]
我也是这种情况，胡言乱语

**Comment by dbian** [comment]
试试这个，[基于gpt-x-vicuna](https://huggingface.co/dbian/gpt4-x-vicuna-13B-chinese)做的，效果还不错

**Reply by CRGBS** [reply]
> 试试这个，[基于gpt-x-vicuna](https://huggingface.co/dbian/gpt4-x-vicuna-13B-chinese)做的，效果还不错

![圖片](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/32331418/a2ad6814-b1ce-4af9-a8e2-6b4f46731203)
感覺往奇怪的方向變糟糕了(環境為llamacpp最新版)

如果只用chinese llama alpaca
![圖片](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/32331418/a2ff7306-afed-4ca0-ae99-030b8c82366e)
就正常了許多

由於我都拿來用代碼 所以只問代碼問題 😂
